src/api/buddies.ts

```diff
@@ -41,10 +50,16 @@ const mappedStatuses: MappedStatuses = {
   Delete: 'deleted',
 };
 
+const mappedFromApi: MappedStatusesFromApi = {
+  banned: 'Banned',
+  deleted: 'Deleted',
+  ok: 'NotBanned',
+};
```

Comment:
as const  here and just drop the declaration :  )

Reply:
Yeah, but now I had to specify these values to be of BanStatus, type. Is it OK?

Reply:
this bad. Now it will break when somebody refactors those names. One should never use as if it is not "as  const"

Reply:
Yeah, i figured. Do you have any suggestion for the obvious fix?

Reply:
`const toBanStatus = {
  banned: 'Banned',
  deleted: 'Deleted',
  ok: 'NotBanned',
} as const;`

Reply:
Oh, you commented it already. Did not register it. Nice trick =)